lib/engine_socketio.js

```diff
@@ -156,7 +153,9 @@ SocketIoEngine.prototype.compile = function (tasks, scenarioSpec, ee) {
       }));
 
     let steps = _.flatten([
-      zero,
+      function z(cb) {
```

Comment:
Here we can use `async.constant(initialContext)` as the first step instead of wrapping the call to `zero` in another function.